src/components/FormMessage.vue

```diff
@@ -192,6 +223,15 @@ export default {
       } else if (this.urlSelected == true) this.text += " <http://url|text>";
     },
 
+
+    preview() {
```

Comment:
I guess this is not done. Avoid commiting stuff that is only started, this is achievable using git